src/ui/public/visualize/loader/pipeline_helpers/run_pipeline.ts

```diff
@@ -20,10 +20,11 @@
 // @ts-ignore
 import { fromExpression } from '@kbn/interpreter/common';
 // @ts-ignore
-import { interpretAst } from '@kbn/interpreter/public';
+import { getInterpreter } from 'plugins/interpreter/interpreter';
```

Comment:
This PR moves some things out of the package and into the plugin; I've been wondering this for a bit, and this seems like a good time to ask. 

What's the divide between `@kbn/interpreter` and the interpreter plugin that ships with Kibana?

Reply:
we try to put everything in a package, but:
- package should have no state, thats why we have the plugins/interpreter ...
it pretty much creates an instance of interpreter, holds its state and shares it with every other plugin. (and serves some routes)